addons/uh60_fms/functions/fnc_interaction_waypoint.sqf

```diff
@@ -31,6 +31,23 @@ switch (_action) do {
         } forEach _microDagrWaypoints;
         _vehicle setUserMFDvalue _value;
     };
+    case "flir": {
```

Comment:
This will conflict with the JVMF branch.
After JVMF is merged we'll also have an addWaypoint function in the fms module and we will be using that in the future for this kind of thing, as we need to make sure all waypoints are also sent to the microdagr waypoint store